ffi/ffi-macros/src/ontology.rs

```diff
@@ -250,6 +348,8 @@ impl Drop for CSlot {
         take_back_c_string!(self.raw_value);
         take_back_c_string!(self.entity);
         take_back_c_string!(self.slot_name);
+        let _ = unsafe { CSlotValue::from_raw_pointer(self.value) };
```

Comment:
use `drop_raw_pointer` instead (avoids having to put a `let _ =` in front of it)